index.html

```diff
@@ -204,25 +203,129 @@ <h2>Mentors for you</h2>
 		                </div>
 		            </div>
 		        </div>
-		    </section>
-                    <section class="bg-secondary">
+			</section>
+			<!--contributors section-->
+			<section>
+
+				<center><h2>Contributors</h2></center>
+		        <div class="col-md-8 col-md-push-2">
+		        <div class="container">
+		        <div class="row">
+				 <p>
+		                Follow contributors on social, simpally click their profile names...
+		        </p>
+		        </div>
+					</div>
+					<div class="row contributors-wrapper">			
+					  <div class="members contributers" id="Contributors">
+						<h1 class="section-h1">
+					  </div>
+							<style>
```

Comment:
Good work, Just one thing. Don't add inline styles. Instead, use CSS classes and stylesheets.

Reply:
ok, I use style sheet

Reply:
contributor.css